### DISCUSSION TITLE
Crash Domoticz with this plugin

### DISCUSSION THREAD

**Discussion by dthery** [addiejanssen/domoticz-solaredge-modbustcp-plugin#40] [Q&A] [answered]
Hi, I am trying to connect my SE7K inverter.
I have opened the port 1502 on the inverter and the ping on it is ok.
But when I set the plugins on Domoticz, the server crashes and I have to reload an old backup.
I am in python 3.9 with pymodbus 2.4.0

Thank you for your help

**Comment by addiejanssen** [comment]
Hi @dthery,

I have no clue how to help you with this if this is the only information available....
Is there anything in the log file?
What do you mean by "the server crashes"?
Please share all the information you have and perhaps we can help.

**Comment by dthery** [comment, marked as answer]
I just solved by installing the beta version of Domoticz.  This is not from your plugin, but from a python problem.
